var/spack/repos/builtin/packages/chai/package.py

```diff
@@ -35,8 +35,12 @@ class Chai(CMakePackage, CudaPackage, ROCmPackage):
 
     depends_on('cmake@3.8:', type='build')
     depends_on('cmake@3.9:', type='build', when="+cuda")
-    depends_on('blt', type='build')
-    depends_on('blt@0.3.7:', type='build', when='+rocm')
+
+    depends_on('blt', type='build', when='chai@2.3.1')
```

Comment:
Should this be chai@2.3.1: (colon at the end, 2.3.1 or after?) That would also mirror what you did with raja more closely.